src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -90,7 +90,7 @@ export const PublishingAccordion = ({
   );
 
   const userCanCreatePublishingRequest = userHasAccessRight(registration, 'publishing-request-create');
-  const userCanApprovePublishingRequest = userHasAccessRight(registration, 'publishing-request-approve');
+  const userCanApprovePublishingRequest = userHasAccessRight(registration, 'publishing-request-approve'); // TODO: This should be handled by allowedOperations (NP-48995)
```

Comment:
Håper dette kan utbedres og kommentaren fjernes før merge
```suggestion
  const userCanApprovePublishingRequest = userHasAccessRight(registration, 'publishing-request-approve');
```